<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the private '.service_locator.XsDp7Mn' shared service.

return $this->privates['.service_locator.XsDp7Mn'] = new \Symfony\Component\DependencyInjection\ServiceLocator(array('servico' => function (): \Novosga\Entity\Servico {
    return ($this->privates['.errored..service_locator.XsDp7Mn.Novosga\Entity\Servico'] ?? $this->load('getServico3Service.php'));
}, 'translator' => function () {
    return ($this->services['translator'] ?? $this->getTranslatorService());
}, 'usuario' => function (): \Novosga\Entity\Usuario {
    return ($this->privates['.errored..service_locator.XsDp7Mn.Novosga\Entity\Usuario'] ?? $this->load('getUsuario2Service.php'));
}));
